src/lib/components/01-elements/img/img.tsx

```diff
@@ -0,0 +1,24 @@
+import Typography from "../typography/typography";
+
+type ImgProps = {
+  imgPath: string;
+  imgDesc?: string;
+  theme?: 'dark' | 'light';
+};
+
+export default function Img({imgPath, imgDesc, theme = 'light'}: ImgProps) {
+  return (
+    <figure>
+      <img src={imgPath} alt={imgDesc} />
+      {imgDesc !== '' && (
+        <figcaption>
+          <Typography
+            variant='body-4'
+            className={theme === 'dark' ? 'text-azure' : 'text-black'}>
```

Comment:
We use `cn` utility function to deal with conditional classes in components, which it' a combination of [clsx](https://www.npmjs.com/package/clsx) and [tailwind merge](https://www.npmjs.com/package/tailwind-merge). 

Essentially the  `cn` function takes 3 parameters:

- `string`: Base Tailwind classes
- `object`: conditional key (tailwind classes) to be apply if the value is true.
-  `string`: ClassNames we want to override from previous 2 parameters. We use this typically to let user add and override tailwind the component. 

Feel free to reach out if you have doubts